backend/src/API/shoutoutAPI.ts

```diff
@@ -42,3 +42,11 @@ export const hideShoutout = async (
   if (!shoutout) throw new NotFoundError(`Shoutout with uuid: ${uuid} does not exist!`);
   await ShoutoutsDao.updateShoutout({ ...shoutout, hidden: hide });
 };
+
+export const deleteShoutout = async (uuid: string, user: IdolMember): Promise<void> => {
+  const shoutout = await ShoutoutsDao.getShoutout(uuid);
+  if (!shoutout) {
+    throw new NotFoundError(`No shoutout with id '${uuid}' found.`);
+  }
+  await ShoutoutsDao.deleteShoutout(uuid);
+};
```

Comment:
Need permissions check

Reply:
Would the permission check in this case be that the user trying to delete the shoutout is the same as the user who created the shoutout?

Reply:
Yeah I think to be able to delete a shoutout, the user needs to have created the shoutout or be a lead/admin

Reply:
That makes sense. Can a lead/admin delete other lead/admin's shoutouts?

Reply:
Hmmm... I don't see why not 🤷🏻‍♂️